README.md

```diff
@@ -33,6 +33,12 @@ app({
   }
 })
 ```
+## Browser Support
+Firefox | Chrome | IE   | Edge | Safari
+-------|---------|------|------|-------
+55 ✔️   | 61 ✔️    | 11 ✔️ | 15 ✔️ | 10 ✔️ 
```

Comment:
Why Safari 10? There is no way that Safari 9 is worse than IE10.

Reply:
I just went with latest everything except known exceptions, I'm sure it works in chrome 35 too. Does anyone have access to safari 9 to test?

Reply:
@VinSpee Could we add this to the readme in the / after or before the installation instructions? 🤔

Reply:
BTW, >=IE10  😄

Reply:
👍🏻 got it to 1 line now

> Could we add this to the readme in the / after or before the installation instructions?

My personal preference is to have it in the repo index - i find it's one of the first questions people will ask when looking for things like this. I'm happy to move it if you like however.

Reply:
@VinSpee Yeah, I understand, but the reason I haven't added it until now is because I haven't been able to find a good spot for it yet.

I cam up with this line:

> Hyperapp supports all ES5-compliant browsers, >=IE10.

But I am not sure where to put in on the README that won't break the balance we have at the moment.